cmd/helm/installer/install.go

```diff
@@ -129,24 +129,23 @@ func generateLabels(labels map[string]string) map[string]string {
 }
 
 // parseNodeSelectors takes a comma delimited list of key=values pairs and returns a map
-func parseNodeSelectors(labels string) map[string]string {
+func parseNodeSelectors(labels string, m map[string]string) map[string]string {
```

Comment:
@seh User needs to be able to override that default selector. Are you suggesting that anything that needs to be mutated needs to be passed as a pointer?